### PR TITLE
Updating hunting queries of UEBA

### PR DESCRIPTION
**Pull request by manishkumar1991**
Required items, please Complete
   
   Change(s):
   - Hunting queries were not updated

   Reason for Change(s):
   - hunting queries updated with latest mapping and minor fixes

   Version Updated:
   - N/A

   Testing Completed:
   - N/A

   Checked that the validations are passing and have addressed any issues that are present:
   - N/A

### CODE REVIEW

File: Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ query: |
   IdentityInfo
   | where AccountCreationTime < ago(7d)
```
Comment (vakohl) [inline Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml:14]: seeing the rule description, it seems like greater than filter should be used.

File: Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ query: |
...
   | where GroupMembership contains adminGroup
+  | extend Account_0_AadUserId = AccountObjectId
 entityMappings:
   - entityType: Account
     fieldMappings:
```
Comment (vakohl) [inline Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml:19]: can we add versions to all Hunting queries? version: 2.0.0

File: Solutions/UEBA Essentials/Hunting Queries/terminatedEmployeeAccessHVA.yaml [hunk 3]
```diff
@@ -9,13 +9,17 @@ requiredDataConnectors:
 tactics:
 relevantTechniques:
 query: |
+  //Consider creating the watchlist named "TerminatedEmployees" and "HighValueAssets" having column "User Principal Name" and "Asset Name" with details of terminated employees
+  //If you already have watch list with diffrent name then consider modifying the query
```
Comment (vakohl) [inline Solutions/UEBA Essentials/Hunting Queries/terminatedEmployeeAccessHVA.yaml:13]: //If you already have watchlist with different name then consider modifying the query

File: Solutions/UEBA Essentials/Hunting Queries/terminatedEmployeeAccessHVA.yaml [hunk 4]
```diff
@@ -9,13 +9,17 @@ requiredDataConnectors:
...
+  _GetWatchlist('TerminatedEmployees')
```
Comment (vakohl) [inline Solutions/UEBA Essentials/Hunting Queries/terminatedEmployeeAccessHVA.yaml:14]: we can remove line 14 as we are joining same watchlist at line 19

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml (hunk 2 above)
Comment (manishkumar1991) [reply Solutions/UEBA Essentials/Hunting Queries/newAccountAddedToAdminGroup.yaml:19]: done

**Review by vakohl — APPROVED** [review]

**Pull request merged**